drivers/storage/nvme/NvmeStd.c

```diff
@@ -0,0 +1,330 @@
+#include "precomp.h"
```

Comment:
I'd suggest to add standard headers per our coding style, at least where there are no header specified.